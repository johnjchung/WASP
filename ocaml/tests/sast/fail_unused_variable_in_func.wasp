int a = 11;
int b = 13;

Func int squareFloat2(int k) {
	int j = a + b + k;
	return a + b;
}
Endpoint e(GET:int x) {
    int c = squareFloat2(x);
}
