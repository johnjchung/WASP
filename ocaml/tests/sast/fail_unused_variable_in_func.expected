Fatal error: exception Symbol_table.UnusedVariableError("THE VARIABLE j WAS NEVER USED")
